src/test/java/games/strategy/engine/framework/startup/login/ClientLoginValidatorTests.java

```diff
@@ -15,26 +16,48 @@
 import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
 
+import com.google.common.base.Strings;
 import com.google.common.collect.ImmutableMap;
 
 import games.strategy.engine.framework.startup.login.ClientLoginValidator.ErrorMessages;
-import games.strategy.net.IServerMessenger;
+import games.strategy.util.MD5Crypt;
+
 
 @RunWith(Enclosed.class)
 public final class ClientLoginValidatorTests {
   private static final String PASSWORD = "password";
   private static final String OTHER_PASSWORD = "otherPassword";
 
+  private static final String MAGIC_MAC_START = MD5Crypt.MAGIC + "MH$";
```

Comment:
To some extent to avoid the duplication between test and prod code,  sure: 0f08b8f4f

I did not extract other constants such as the regex, not much value in a magic constant over a magic string if the magic string is used in exactly one spot.